arviz/data/inference_data.py

```diff
@@ -417,6 +448,39 @@ def stack(
             A new InferenceData object by default.
             When `inplace==True` perform selection in-place and return `None`
 
+        Examples
+        --------
+        Use ``stack`` to stack any number of existing dimensions into a single new dimension.. We first check the
```

Comment:
Using some example to stack chain and draw dims into `sample` one will probably be easier to follow and understand for users